client/components/Menu/Menu.jsx

```diff
@@ -36,51 +20,45 @@ const styles = theme => ({
   },
   appBar: {
     marginLeft: drawerWidth,
-    [theme.breakpoints.up('sm')]: {
-      width: `calc(100% - ${drawerWidth}px)`,
-    },
+    [theme.breakpoints.up('sm')]: { width: `calc(100% - ${drawerWidth}px)` },
   },
   menuButton: {
     marginRight: 20,
-    [theme.breakpoints.up('sm')]: {
-      display: 'none',
-    },
+    [theme.breakpoints.up('sm')]: { display: 'none' },
   },
   toolbar: theme.mixins.toolbar,
-  drawerPaper: {
-    width: drawerWidth,
-  },
+  drawerPaper: { width: drawerWidth },
   content: {
     flexGrow: 1,
     padding: theme.spacing.unit * 3,
-    textAlign: "center",
+    textAlign: 'center',
   },
-  grow: {
-    flexGrow: 1,
-  },
-  languagebar: {
-    textAlign: "center",
-  }
+  grow: { flexGrow: 1 },
+  languagebar: { textAlign: 'center' },
 });
 
 class ResponsiveDrawer extends React.Component {
-  state = {
-    mobileOpen: false,
-  };
+  constructor(props) {
+    super(props);
+    this.state = { mobileOpen: false };
+  }
 
```

Comment:
I think this was why DevLee wanted the class-prop plugin, can set state without using the constructor.